src/views/components/driverCard.js

```diff
@@ -11,6 +12,13 @@ function Driver({ user, dataTestID }) {
         setShowChatModal(false);
     };
 
+    const { banUserResult, handleBanUser, banUserError } = useBanUser();
```

Comment:
We should have [Alerts](https://react-bootstrap.netlify.app/docs/components/alerts/) to specify this ban operation succeeded or errored